util/src/main/resources/org/killbill/billing/util/migration/V20200916171757__recreate_users_index_as_unique.sql

```diff
@@ -0,0 +1,2 @@
+drop index users_username on users;
+create unique inex users_username ON users(username);
```

Comment:
Typo: `inex`.

Could you run it, make sure it works?

Reply:
Apologies, I did run it and noticed the typo, but forgot had already committed. My bad.

```sql
> drop index users_username on users;
Query OK, 0 rows affected (0.009 sec)
Records: 0  Duplicates: 0  Warnings: 0

> create unique index users_username ON users(username);
Query OK, 0 rows affected (0.015 sec)
Records: 0  Duplicates: 0  Warnings: 0

> show index from users;
+-------+------------+----------------+--------------+-------------+-----------+-------------+----------+--------+------+------------+---------+---------------+
| Table | Non_unique | Key_name       | Seq_in_index | Column_name | Collation | Cardinality | Sub_part | Packed | Null | Index_type | Comment | Index_comment |
+-------+------------+----------------+--------------+-------------+-----------+-------------+----------+--------+------+------------+---------+---------------+
| users |          0 | PRIMARY        |            1 | record_id   | A         |           0 |     NULL | NULL   |      | BTREE      |         |               |
| users |          0 | record_id      |            1 | record_id   | A         |           0 |     NULL | NULL   |      | BTREE      |         |               |
| users |          0 | users_username |            1 | username    | A         |           0 |     NULL | NULL   | YES  | BTREE      |         |               |
+-------+------------+----------------+--------------+-------------+-----------+-------------+----------+--------+------+------------+---------+---------------+
3 rows in set (0.001 sec)
```